lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +240,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        link_tags = article_bs.find_all('a')
```

Comment:
how do you know that you do not collect links to external websites, for example, links from advertisement?

Reply:
Changed it